### PR TITLE
[CIVIS-8783] Add Github Actions manual approval feature to docs

### PR DESCRIPTION
**Pull request by rodrigo-roca**
### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/continuous_integration/pipelines/github.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ further_reading:
 
 - **Queue time**: View amount of time workflow jobs sit in the queue before processing
 
+- **Approval wait time**: View amount of time workflow runs and workflow jobs wait for a manual approval
```
Comment (andrea-mosk) [inline content/en/continuous_integration/pipelines/github.md:42]:
```suggestion
- **Approval wait time**: View amount of time workflow runs and workflow jobs wait for manual approvals
```

File: content/en/continuous_integration/pipelines/_index.md [hunk 2]
```diff
@@ -155,21 +155,22 @@ If your CI provider is not supported, you can try setting up Pipeline Visibility
...
+| {{< ci-details title="Parameters" >}}Support for adding custom pipeline parameters that users set (for example, <code>DYNAMICS_IS_CHILD:true</code>). You can then search using these parameters in the <a href="https://docs.datadoghq.com/continuous_integration/explorer/?tab=pipelineexecutions">CI Visibility Explorer</a> to find all events with a specific parameter.{{< /ci-details >}} |                  {{< X >}}                   | {{< X >}} |           |           |                |                 | {{< X >}} |           |                   |     {{< X >}}      |
+| {{< ci-details title="Pipeline failure reason" >}}Identification of a specific reason behind a pipeline or job failure.{{< /ci-details >}}                                                                                                                                                                                                                                                       |                  {{< X >}}                   | {{< X >}} |           |           |                |                 | {{< X >}} | {{< X >}} |     {{< X >}}     |     {{< X >}}      |
```
Comment (andrea-mosk) [inline content/en/continuous_integration/pipelines/_index.md:174]: Let's keep the same syntax as before? I would expect just a {{< X >}} change here

Comment (rodrigo-roca) [reply content/en/continuous_integration/pipelines/_index.md:174]: oh crap my IDE did some nasty things

**Review by andrea-mosk — APPROVED** [review]

File: content/en/continuous_integration/pipelines/_index.md [hunk 3]
```diff
@@ -161,8 +161,9 @@ If your CI provider is not supported, you can try setting up Pipeline Visibility
...
 | {{< ci-details title="Step spans" >}}Step level spans are available for more granular visibility.{{< /ci-details >}} | {{< X >}} (_But are presented as job spans_) | {{< X >}} |  |  |  |  | {{< X >}} |  |  |  {{< X >}} |
-| {{< ci-details title="Manual steps" >}}Identification of when there is a job with a manual approval phase in the overall pipeline.{{< /ci-details >}} | {{< X >}} | {{< X >}} |  | {{< X >}} |  |  | {{< X >}} |  |  |  {{< X >}} |
+| {{< ci-details title="Manual steps" >}}Identification of when there is a job with a manual approval phase in the overall pipeline.{{< /ci-details >}} | {{< X >}} | {{< X >}} |  | {{< X >}} | {{< X >}} |  | {{< X >}} |  |  |  {{< X >}} |
 | {{< ci-details title="Queue time" >}}Identification of the amount of time for which a pipeline or job was in the queue before execution.{{< /ci-details >}} | {{< X >}} | {{< X >}} |  | {{< X >}} | {{< X >}} |  | {{< X >}} | {{< X >}} |  |  {{< X >}} |
+| {{< ci-details title="Approval wait time">}}Identification of the amount of time for which a pipeline or job have been waiting for a manual approval.{{< /ci-details >}} |  |  |  |  |  {{< X >}}  |   |   |  |  |  |
```
Comment (alai97) [inline content/en/continuous_integration/pipelines/_index.md:166]:
```suggestion
| {{< ci-details title="Approval wait time">}}Identification of the amount of time for which a pipeline or job has been waiting for a manual approval.{{< /ci-details >}} |  |  |  |  |  {{< X >}}  |   |   |  |  |  |
```

**Review by alai97 — APPROVED** [review]
LGTM

**Comment by alai97** [comment]
Feel free to ping me on Slack or leave a comment here when you want this merged @rodrigo-roca

**Comment by rodrigo-roca** [comment]
> Feel free to ping me on Slack or leave a comment here when you want this merged @rodrigo-roca

@alai97 Now this is ready to merge, thanks for the review! I just moved up the row I added one position up.

**Pull request merged**